### ISSUE TITLE
Removed feedback tags still appear in message response views until another feedback tag is edited or added

### ISSUE THREAD

**Issue by etanttila** [apluslms/mooc-jutut#55]
If one removes a feedback tag on the "Manage feedback tags" page, the taggings from the feedback are removed as they should as well as the tag from the filter view. However, it still appears as grey in the message response views.
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/006b516f-1a16-4fba-90fb-16628bfd0651)
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/625150cc-d815-45d2-b55b-61da6a3b6497)
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/f03a9547-8620-4fb4-8551-fd9b07026053)
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/76abd2bc-f518-470a-a3fd-ec8834a7ebee)

If one tries to select the tag, it informs of the error (as the tag doesn't exist, so this is good behavior).
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/4f29d1d6-4e44-4d42-b9b5-98ae1e67f783)

Even if the page is refreshed or Jutut is closed and reopended, the removed tag still appears. 
The removed tag disappears from the messages only when another feedback tag is edited or added OR the cache is cleared (with the Clear cache" option in the menu).
Basically it seems like the problem is that the message response views aren't updated if a feedback tag is removed.

**Comment by Mankro** [comment]
If the solution is to clear the cache, then it could be possible to clear the cache when the tag is removed. Cache may be cleared in the view class code, but it is also possible to connect signal handlers to the model's delete operation so that the cache is always cleared when the object's `delete()` method is called.
